app/src/main/java/com/gophillygo/app/data/models/CategoryAttraction.java

```diff
@@ -0,0 +1,94 @@
+package com.gophillygo.app.data.models;
+
+import android.util.SparseArray;
+
+import java.util.Objects;
+
+import static com.gophillygo.app.data.models.Filter.EDUCATIONAL_CATEGORY;
+import static com.gophillygo.app.data.models.Filter.EXERCISE_CATEGORY;
+import static com.gophillygo.app.data.models.Filter.NATURE_CATEGORY;
+
+/**
+ * Model for the home screen categories, which display a random representative image from
+ * places of a given category.
+ */
+public class CategoryAttraction {
+
+    // category grid cards that are not in the destination category string: two user flags,
+    // and events
+    public static final String UPCOMING_EVENTS = "Upcoming events";
+    public static final String WANT_TO_GO = "Want to go";
+    public static final String LIKED = "Places you like";
+
+
+    // FIXME: reference user-facing strings (displayName) as strings resources
```

Comment:
It would be nice to get this cleaned up as part of this PR.

Instead of a string field in the enum options, we could replace it with a `@StringRes int` field and leave it to the user of the enum to turn that into an actual string as-needed.

Probably fine to continue to using string constants for the `dbName` field.